WebPhapp/backend/mysql_helper.js

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Why was this created as a separate file? I am not complaining, because I think separating the SQL queries is a good idea, but I am curious why.